Makefile

```
@@ -66,15 +67,19 @@ ifndef IMAGE_REGISTRY_USERNAME
 endif
 ifndef IMAGE_REGISTRY_PASSWORD
 	$(error IMAGE_REGISTRY_PASSWORD is undefined)
+endif
+ifdef SELF_SIGNED_CERT
+	oc create configmap self-signed-cert --from-file=ca-bundle.crt=${SELF_SIGNED_CERT}
 endif
 	@sed -e "s#{{ AWS_SECRET_ACCESS_KEY }}#${AWS_SECRET_ACCESS_KEY}#g" -e "s#{{ AWS_ACCESS_KEY_ID }}#${AWS_ACCESS_KEY_ID}#g" -e "s#{{ S3_REGION }}#${S3_REGION}#g" -e "s#{{ S3_ENDPOINT }}#${S3_ENDPOINT}#g" pipelines/tekton/aiedge-e2e/templates/credentials-s3.secret.yaml.template | oc create -f -
 	@sed -e "s#{{ IMAGE_REGISTRY_USERNAME }}#${IMAGE_REGISTRY_USERNAME}#g" -e "s#{{ IMAGE_REGISTRY_PASSWORD }}#${IMAGE_REGISTRY_PASSWORD}#g" pipelines/tekton/aiedge-e2e/templates/credentials-image-registry.secret.yaml.template | oc create -f -
 	oc secret link pipeline credentials-image-registry
 
 # requires:
-#	S3_BUCKET		- Name of S3 bucket that has the model
-#	NAMESPACE		- Cluster namespace that tests are run in
+#	S3_BUCKET	- Name of S3 bucket that contains the models
+#	NAMESPACE	- Cluster namespace that tests are run in
 #	TARGET_IMAGE_TAGS_JSON	- JSON array of image tags that the final image will be pushed to. E.g. '["quay.io/user/model-name:e2e-test"]'
+#	SELF_SIGNED_CERT	- Local path to the self signed cert to be used in pipeline (optional)
```

I was thinking that this may be aligned as well, but that is up to you.
```suggestion
#	S3_BUCKET	        - Name of S3 bucket that contains the models
#	NAMESPACE	        - Cluster namespace that tests are run in
#	TARGET_IMAGE_TAGS_JSON	- JSON array of image tags that the final image will be pushed to. E.g. '["quay.io/user/model-name:e2e-test"]'
#	SELF_SIGNED_CERT	- Local path to the self signed cert to be used in pipeline (optional)
```

---

Sure no problem